lib/constants.ts

```diff
@@ -250,8 +250,20 @@ export enum CYCLE {
 
 export const BLACK_FRIDAY = {
     COUPON_CODE: 'BF2019',
-    START: 'November 26, 2019 06:00:00',
-    END: '2019-12-09'
+    START: {
+        // Nov 26, 2019 06:00:00 UTC
+        YEAR: 2019,
+        MONTH: 10, // An integer between 0 (January) and 11 (December) representing the month
+        DAY: 26,
+        HOUR: 6
```

Comment:
Unless you have a need for these variables I think it's fine if you declare the date already here

```suggestion
        START: new Date(Date.UTC(2019, 10, 26, 6));
```